content/docs/how-we-work/accessibility.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Accessibility
+weight: 9
+description: We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+---
+
+# Accessibility
+
+We implement 508 and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
```

Comment:
> whether it be physical, mental, or visual impairment

Is this official verbiage? I don't know how WCAG addresses mental impairment. Further, isn't visual impairment physical in nature?

Reply:
Let's use the term [neurodivergence](https://wcag.com/blog/digital-accessibility-and-neurodiversity/) instead.

Reply:
https://axelerant.atlassian.net/wiki/spaces/AH/pages/4040824379/Accessibility

Reply:
@hussainweb, made as per comments and naming changes as per WCAG.